MIGRATION.md

```diff
@@ -262,153 +265,14 @@
 
 ## From version 6.5.x to 7.0.0
 
-### Alpha release notes
-
-Storybook 7.0 is in early alpha. During the alpha, we are making a large number of breaking changes. We may also break the breaking changes based on what we learn during the development cycle. When 7.0 goes to beta, we will start stabilizing and adding various auto-migrations to help users upgrade more easily.
-
-In the meantime, these migration notes are the best available documentation on things you should know upgrading to 7.0.
+A number of these changes can be made automatically by the storybook cli. To take advantage of these "automigrations", run `npx storybook@next upgrade --prerelease` or `pnpx storybook@next upgrade --prerelease`.
```

Comment:
@IanVS can you capitalize both Storybook and CLI, turning this into: "A number of these changes can be made automatically by the Storybook CLI."